examples/dt/compute/compute-starter-kit/control-plane.md

```diff
@@ -0,0 +1,66 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `crc-csi-hostpath-provisioner` should already exist.
+
+## Initialize
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the hci directory
```

Comment:
Wouldn't this be change to the compute-starter-kit directory?

Reply:
yep
this file is mostly just a copy past right now but ill correct this when i get to the datapalne part

this dt will have at least the following phases.

deploy openstack with nova and cell0
deploy cell1
 - first deploy the contolplane part of cell1 and then waith for ti to be readay
 - then deploy the dataplane
finally perfrom post install config (flavor/image creations) and run tests.